x/chat/types/keys.go

```diff
@@ -22,7 +22,7 @@ const (
 	ChannelCountKey = "channel_count"
 
 	// ChannelKey is the key to store the channels
-	ChannelKey = "channel"
+	ChannelKey = "channel_store"
```

Comment:
Is there a reason, why we can't use `channel-` as a prefix?

Reply:
That would work, you just need to have a name that is not a prefix of another key name